<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serial</class>
 <widget class="QWidget" name="Serial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="serialBox">
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>30</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Speed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Parity</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Bits</string>
      </property>
     </widget>
     <widget class="QComboBox" name="portComboBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="speedComboBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="dataComboBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="parityComboBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>none</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>odd</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>even</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="stopbitsComboBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="serialstartPushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>bits</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>230</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>bits</string>
      </property>
     </widget>
     <widget class="QPushButton" name="serialstopPushButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
